### DISCUSSION TITLE
Should days of prayer be inserted into particular calendars?

### DISCUSSION THREAD

**Discussion by tukusejssirs** [romcal/romcal#382] [Q&A]
Related: #110

---

I am a dilemma if we should add days of prayer into particular calendars if those prayer are inscribed into a particular calendar (and therefore listed in the proper of the particular region/country).

For example, in the US, there celebrate `Day of Prayer for the Legal Protection of Unborn Children` on 22 January (or 23 January if 22 January is Sunday):
- it is listed in the Proper Calendar for the Dioceses of the United States of America [here](https://www.usccb.org/prayer-and-worship/liturgical-year-and-calendar/proper-calendar) and [here](https://www.usccb.org/resources/2020cal.pdf#page=10);
- in the US [_Ordo 2020_](https://www.usccb.org/resources/2020cal.pdf#page=16), they list a different colour (purple) as well as the readings are (or might be?) used different for the celebration of DPLPUC.

Similar celebrations can be found in the Propers of Australia and New Zealand

**Comment by tukusejssirs** [comment]
Notes to myself in case they will be implemented:

```js
day_of_prayer_for_the_legal_protection_of_unborn_children: 'Day of Prayer for the Legal Protection of Unborn Children',  // en
day_of_prayer_for_the_legal_protection_of_unborn_children: 'Dies orationis pro protectio legalis liberorum non natorum',  // la; TODO: This needs to be proofed
day_of_prayer_for_the_legal_protection_of_unborn_children: 'Deň modlitby za zákonnú ochranu nenarodených detí',  // sk
```

**Comment by tukusejssirs** [comment]
List of such celebrations from the US proper:
- Day of Prayer for the Legal Protection of Unborn Children (22 Jan; it is moved to 23 Jan, when 22 Jan falls on a Sunday);
- Independence Day (4 July);
- Thanksgiving Day (fourth Thursday in November).

List of such celebrations from the Australian proper:
- Anzac Day (25 April).

List of such celebrations from the proper of New Zealand:
- Waitangi Day (6 February);
- Anzac Day (25 April).

List of such celebrations from the Canadian proper ([src](https://www.cccb.ca/wp-content/uploads/2017/09/nationallitcalender2016_01_27.pdf)):
- Canada Day (1 July);
- Labour Day (first Monday of September; don’t forget to localise it as `Labor Day` in `en.ts`);
- Thanksgiving Day (second Monday of October);
- Remembrance Day (11 October).

**Comment by BillTheGoat** [comment]
Since the goal is to reproduce the calendars approved by the Church, it seems clear they should be included.

**Comment by tukusejssirs** [comment]
@BillTheGoat, I think so too, however, I don’t want to add something we don’t agree on beforehand. I am still waiting on @emagnier and his opinion (he is probably busy with work currently).

**Comment by tukusejssirs** [comment]
At least for the US days, according to [this conversation](https://en.m.wikipedia.org/wiki/Talk:General_Roman_Calendar#Changes_by_Jrcovert), they definitely should be included in romcal.

**Comment by tukusejssirs** [comment]
My current opinion on this is that if a particular day of prayer is inscribed in a particular calendar, it should be in romcal. Otherwise, they should not be part of romcal (which should include only the GRC, and particular calendars which are needed to generate official Ordos and the calendars in the Roman Missal official translations). Other days of prayers could be part of parish calendars, which should not be part of romcal, but they could will be part of separate repositories (or maybe a separate packages; @emagnier might have a better idea).
